### DISCUSSION TITLE
How can I get the calendar to take up the width of the screen?

### DISCUSSION THREAD

**Discussion by PavelGutin** [eouia/MMM-CalendarExt2#290] [Q&A]
Thank you for a great module. Is there an easy to way to get the calendar to take up the full width? Here's my config, and it's only spanning 700 pixels

```js
 {
            module: 'MMM-CalendarExt2',  
            position: "bottom_bar",
            config: {
              calendars : [
                  {
                    name: "School",
                    url: "https://calendar.google.com/....."
                  },
                  {
                    name: "Family",
                    url: "https://calendar.google.com/...."
                  }
              ],
              views: [
                {
                  mode: "week",
                },
              ],
              scenes: [
                {
                  name: "DEFAULT",
                },
              ],
            },
          },
```

**Comment by KristjanESPERANTO** [comment]
You can override the CSS custom property that limits the column width. Add this to your `css/custom.css`:

```css
.CX2 {
  --column-max-width: none;
}
```

This removes the maximum width cap and lets the week view expand to fill the available space. You can also set a specific pixel value instead of `none` if you want a defined width.

See the [Styling documentation](https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/blob/main/docs/Styling.md) for more available CSS custom properties.
